xpdacq/glbl.py

```diff
@@ -137,24 +137,35 @@ class Glbl:
                  'tri_offset': 13, 'v_asym': 0,
                  'alpha': 2.5, 'tmsk': None}
 
-    def __init__(self, frame_acq_time=FRAME_ACQUIRE_TIME):
-        self._frame_acq_time = frame_acq_time
+    def __init__(self, filepath, internal_dict=None):
+        if internal_dict is None:
+            internal_dict = {}
+        super().__init__(internal_dict)
+        self._filepath = filepath
+
+    def allowed_attributes(self):
+        """list of attribute that are allowed to be serialized"""
+        return ['auto_dark', 'dk_window', '_dark_dict_list',
```

Comment:
I want to prevent it from being updated. If it is a class variable, user could potentially update it (and break it)